wrappers/services.go

```diff
@@ -824,6 +823,32 @@ func ServicesEnableState(s *snap.Info, inter interacter) (map[string]bool, error
 	return snapSvcsState, nil
 }
 
+// RemoveQuotaGroup ensures that the slice file for a quota group is removed. It
+// only works on groups that have no sub-groups, in order to remove a parent
+// group with sub-groups, one must remove all the sub-groups first.
+func RemoveQuotaGroup(grp *quota.Group, inter interacter) error {
+	// TODO: it only works on leaf sub-groups currently
+	if len(grp.SubGroups) != 0 {
```

Comment:
So your recommendation is to not do this check here and leave the existing check in `servicestate.RemoveQuota()`?

Reply:
no, my recommendation is to turn the error here into an "internal error", and also probably that the doc comment should be clear about what this can do and what it isn't aware of: something like

// ... This function assumes that the slice corresponding to the group is not in use anymore by any services when it is invoked.  It does not check for this.